doit.go

```diff
@@ -207,7 +208,7 @@ func (c *LiveConfig) GetGodoClient(trace bool, accessToken string) (*godo.Client
 }
 
 func userAgent() string {
-	return "doctl/" + DoitVersion.String()
+	return fmt.Sprintf("doctl/%s-%s/%s", runtime.GOOS, runtime.GOARCH, DoitVersion.String())
```

Comment:
```suggestion
	return fmt.Sprintf("doctl/%s (%s %s)", DoitVersion.String(), runtime.GOOS, runtime.GOARCH)
```

I think the version should come before the platform. Browsers tend to use a format like: `<product>/<version> (<platform>)`

See: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Reply:
That sounds okay to me.

I'd originally had it as `<product>/<version>/<platform>` to match the file format for Go binary downloads, but @zachgersh suggested the format in this PR. I'm happy to do anything, so if other folks are happy with your suggestion, I'll do that 🙂

Reply:
if only @andrewsomething had changed the test as well 💃

Reply:
If only @andrewsomething was better with regexp!